src/extension.ts

```diff
@@ -126,6 +139,18 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
   disposable.forEach((value) => extensionContext.subscriptions.push(value));
 }
 
+async function sendVersionToTelemetry(commandId: string, cmd: string): Promise<void> {
+  const result = await executeCmdCli.execute(createCliCommand(cmd, 'version'));
+  if (result.error) {
+    telemetryLogError(commandId, result.error);
+  }
```

Comment:
Shouldn't you return/exit after the `telemetryLogError`? If it happens i guess there is no version to report.

Reply:
this is telemetry specific so if there is an error we can see it in the telemetry log.

Reply:
yeah i was just thinking that if result.error has content, the result.stdout will likely not have the version in it so the following `telemetryLog` would make no sense.

Reply:
thanks I have fixed :)